src/output-json-flow.c

```diff
@@ -261,6 +261,12 @@ static void JsonFlowLogJSON(JsonFlowLogThread *aft, json_t *js, Flow *f)
     json_object_set_new(hjs, "reason",
             json_string(reason));
 
+    uint8_t has_alerts = 0;
```

Comment:
How about make this a bit more compact?
e.g.
uint8_t has_alerts = FlowHasAlerts(f);

Or even:
json_object_set_new(hjs, "has_alerts", json_boolean(FlowHasAlerts(f)));